### DISCUSSION TITLE
11월 19일 (금)

### DISCUSSION THREAD

**Discussion by pkwon35** [InChil2/Detection_of_lesions#12] [회의록]
11월 19일 회의록

**Comment by pkwon35** [comment]
팀내에서 각각 돌려볼 모델 나눔. 일단 불균형 데이터셋 활용해서 

FasterRCNN : 권준기, 우동주
EfficientNet : 신인철, 이민찬
Yolo5 : 박시우, 최한결

**Comment by InChil2** [comment]
결과 = EfficientDet 예제나 Usecase가 없어서 Yolov5 사이즈별 테스트 진행
